src/mongo/db/storage/hse/src/hse_durability_manager.h

```diff
@@ -89,4 +93,25 @@ class KVDBDurabilityManager {
     mutable stdx::condition_variable _syncDoneCV;
 };
 
+class KVDBJournalFlusher : public BackgroundJob {
+public:
+    explicit KVDBJournalFlusher(KVDBDurabilityManager& durabilityManager);
+
+    virtual string name() const;
+
+    virtual void run();
+
+    void shutdown();
+
+    void notifyFlusher();
+
+private:
+    KVDBDurabilityManager& _durabilityManager;
+    std::atomic<bool> _shuttingDown{false};  // NOLINT
+
+    bool _flushPending;
+    mutable std::mutex _jFlushMutex;
+    mutable stdx::condition_variable _jFlushCV;
```

Comment:
consider using std::, since we require a c++11 compiler. I know there are other places that use stdx, we need to clean up.

Reply:
We can probably do in this a separate clean up pass :-)